src/BloomExe/CollectionCreating/CollectionNameControl.cs

```diff
@@ -75,9 +75,7 @@ private bool GetNameIsOk()
 				}
 
 				_collectionInfo.PathToSettingsFile = CollectionSettings.GetPathForNewSettings(_destinationDirectory, _collectionNameControl.Text);
-				if (DestinationAlreadyExists)
-					return false;
-				return true;
+				return !DestinationAlreadyExists && _collectionNameControl.Text.ToLowerInvariant() != "templates";
```

Comment:
@hatton, should we give the user some explanation of why his button is disabled? Not that we're doing it if he types the name of an existing collection, I guess.